packages/cds-odata-annotation-converter/test/testUtils/file-access.ts

```diff
@@ -0,0 +1,26 @@
+import { join } from 'path';
+import * as fs from 'fs';
+import type { File } from '@sap/ux-cds-compiler-facade';
+
+export const readFile = (path: string): Promise<string> => {
```

Comment:
This can be replaced by `fs.promises.readFile`.

Reply:
replaced.